### PR TITLE
Action chains fixes

### PR DESCRIPTION
**Pull request by Bischoff**
## What does this PR change?

This PR fixes two things:
 - it removes a hack that was needed at some point to have the minion reboot
 - it makes sure suse manager's packages cache is refreshed before
   we launch the second series of action chains tests

Explanation of the second part:

We were doing this:
* remove andromeda-dummy with zypper
* install andromeda-dummy version 1.0 with zypper
* this will trigger a Package List Refresh event
* we wait for events history to be empty, hoping to wait for the end of that refresh

What could go wrong:
* zypper did not make it in time to trigger the package list refresh
* we see an empty events list at that point, and continue
* and we still see version 2.0 of andromeda-dummy in the UI -> failure

## Links

Ports:
* 3.2: SUSE/spacewalk#10169
* 4.0: SUSE/spacewalk#10168

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

### CODE REVIEW

File: testsuite/features/secondary/min_action_chain.feature [hunk 1]
```diff
@@ -238,18 +237,20 @@ Feature: Action chain on salt minions
...
-    And I enter "andromeda-dummy" in the css "input[placeholder='Filter by Package Name: ']"
-    And I click on the css "button.spacewalk-button-filter" until page does contain "andromeda-dummy-1.0" text
+
+  Scenario: Refresh package list and check installed packages after second downgrade on SLE minion
+    When I refresh packages list via spacecmd on "sle-minion"
+    And I wait until refresh package list on "sle-minion" is finished
+    Then spacecmd should show packages "milkyway-dummy andromeda-dummy-1.0" installed on "sle-minion"
+
+  Scenario: Ensure again the errata cache is computed before testing on Salt minion
+    Given I am on the Systems overview page of this "sle-minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/min_action_chain.feature]: I'm wondering if we need that first step :thinking: 
Looks we jump to the Left Menu option after it

Comment (Bischoff) [reply testsuite/features/secondary/min_action_chain.feature]: Maybe, yes, trying without it.

Comment (Bischoff) [reply testsuite/features/secondary/min_action_chain.feature]: Done.

Comment (Bischoff) [reply testsuite/features/secondary/min_action_chain.feature]:
```    When I follow the left menu "Admin > Task Schedules"                                                 # features/step_definitions/navigation_steps.rb:249
      Unable to find visible xpath "//aside/div[@id='nav']/nav/ul/li/div/a[contains(.,'Admin')]/parent::div/parent::li" (Capybara::ElementNotFound)
```

it looks like ti was needed...

Comment (Bischoff) [reply testsuite/features/secondary/min_action_chain.feature]: Readded, and works now @srbarrios .

**Comment by Bischoff** [comment]
tested 5 times in a row, and no failure, will merge

**Pull request merged**